.travis.yml

```diff
@@ -2,6 +2,8 @@ language: java
 before_cache:
   - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
   - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+before_install:
+  - sudo apt-get update && sudo apt-get install libsodium-dev;
```

Comment:
This is the line to make sure the libsodium library is available for the JNA to do its access.